pkg/didcomm/common/service/destination.go

```diff
@@ -24,7 +24,8 @@ type Destination struct {
 
 const (
 	didCommServiceType = "did-communication"
-	ed25519KeyType     = "Ed25519VerificationKey2018"
+	// TODO: hardcoded key type
```

Comment:
Is there more context on this TODO? Why hardcoded?

Reply:
Figured out that this is not necessary at all, we want all keys in a DID doc to point to the DID, so we don't want to filter at all, here.

Reply:
When we're using another key type for the communication, we need to remove the hardcoding of the key type and instead set it based on some connection parameters.

Reply:
created issue #1008